master/pkg/tasks/task_spec.go

```diff
@@ -238,7 +239,9 @@ func (g GCCheckpoints) Entrypoint() []string {
 
 // Environment implements InnerSpec.
 func (g GCCheckpoints) Environment(TaskSpec) model.Environment {
-	return g.ExperimentConfig.Environment
+	env := model.DefaultExperimentConfig(&g.TaskContainerDefaults).Environment
```

Comment:
Uses `TaskContainerDefaults.Image` if it's there, otherwise our default images, in the event of an airgapped environment with task container defaults set.